README.md

```diff
@@ -78,13 +78,13 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
```

Comment:
consider also whether the React frontend instructions for the project also need to be adapted in the same way as described in the comment below:

- https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/files/77d41f40d80c87a81dbc500e1290a19c4b8128a8#r1598275290

(for external access to the publicly-deployed URL in a Shared / non-private Devbox - may also need to be non-Draft?)

Reply:
We decided offline to remove the Acceptance Criteria for CodeSandbox, and use Drone to create a comment with the CodeSandbox link.